UPGRADING.md

```diff
@@ -1,316 +1,277 @@
+or
 ```swift
-  let config = BugsnagConfiguration("YOUR API KEY HERE")
+Bugsnag.start()
 ```
 
-#### Additions
-
-* `Bugsnag.setBreadcrumbCapacity()` is now `config.setMaxBreadcrumbs()`
+With an entry in your `plist` file:
 
-```diff
-- Bugsnag.setBreadcrumbCapacity(40)
-  let config = try BugsnagConfiguration("YOUR API KEY HERE")
-+ config.setMaxBreadcrumbs(40)
-+ config.persistUser
-
-+ config.persistUserData()
-+ config.deletePersistedUserData()
-
-+ config.addOnBreadcrumb(block:)
-+ config.removeOnBreadcrumb(block:)
-
-+ config.redactedKeys
-+ config.sendThreads
+```xml
+<key>bugsnag</key>
+<dict>
+    <key>apiKey</key>
+    <string>YOUR-API-KEY</string>
+</dict>
 ```
 
-#### Renames
-
-```diff
-- config.autoNotify
-+ config.autoDetectErrors
+You can add further configuration options to the `plist` or construct a `BugsnagConfiguration` to set further options in code. For full details, see the online docs
+([iOS](https://docs.bugsnag.com/platforms/ios/configuration-options/#setting-configuration-options) |
+ [macOS](https://docs.bugsnag.com/platforms/macos/configuration-options/#setting-configuration-options) |
+ [tvOS](https://docs.bugsnag.com/platforms/tvos/configuration-options/#setting-configuration-options)).
 
-- config.autoCaptureSessions
-+ config.autoTrackSessions
-
-- config.beforeSendBlocks
-- config.add(beforeSend:)
-+ config.onSendBlocks
-+ config.addOnSendError(block:)
-
-- config.beforeSessionBlocks
-- config.add(beforeSession:)
-+ config.onSessionBlocks
-+ config.addOnSession(block:)
-
-- config.automaticallyCollectBreadcrumbs
-+ config.enabledBreadcrumbTypes
+#### Additions
 
-- config.reportOOMs
-+ config.enabledErrorTypes
+The following functions have been added to the `Bugsnag` client: 
 
-- config.currentUser
-+ config.user
+| Property/Method                                                    | Usage                                                             |
+| ------------------------------------------------------------------ | ----------------------------------------------------------------- |
+| `addOnBreadcrumb` / `removeOnBreadcrumb`<br />`addOnBreadcrumbBlock` / `removeOnBreadcrumbBlock` | Add/remove callbacks to modify or discard breadcrumbs before they are recorded. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-breadcrumbs/#discarding-and-amending-breadcrumbs) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-breadcrumbs/#discarding-and-amending-breadcrumbs) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-breadcrumbs/#discarding-and-amending-breadcrumbs)).
+| `addMetadata:metadata:toSection` / `addMetadata(metadata:section)` | Adds a dictionary of metadata to a section/tab. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#global-metadata) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#global-metadata) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#global-metadata)).
```

Comment:
section/tab -> section

Reply:
It was referred to as a tab in v5, so wanted to keep something of the old naming. Reworded now for clarity.